packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -283,6 +272,11 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 		},
 	},
 	actions: {
+		getPinDataSize(pinData: Record<string, unknown> = {}): number {
+			return Object.values(pinData).reduce<number>((acc, value) => {
+				return acc + stringSizeInBytes(value as IDataObject);
```

Comment:
Types are a bit of a tangle here but it might not be worth fixing now.

Reply:
Fixed types here.